applications/system-service/wpa_supplicant.h

```diff
@@ -388,7 +388,7 @@ public Q_SLOTS: // METHODS
         return asyncCallWithArgumentList(QStringLiteral("SetPKCS11EngineAndModulePath"), argumentList);
     }
 
-    inline QDBusPendingReply<QVariantMap> SignalPoll()
+    inline QDBusPendingReply<QVariant> SignalPoll()
```

Comment:
Ditto

Reply:
This file is auto generated from the xml file.